File: pinyougou_user_web/src/main/webapp/js/service/userService.js
app.service("userService",function ($http) {


    this.register=function (entity,htmlCode) {
        return $http.post("../user/register.do?htmlCode="+htmlCode,entity)
    };


    this.createCode=function (phone) {
        return $http.get("../user/createSmsCode.do?phone="+phone)
    }
});

File: pinyopuogou_user_interface/src/main/java/com/pinyougou/service/UserService.java
package com.pinyougou.service;

import com.pinyougou.pojo.TbUser;

public interface UserService {


    /**
     * 用户注册
     * @param user
     */
    public void register(TbUser user);

    /**
     * 创建验证码
     * @param phone
     */
    void createSmsCode(String phone);
}
